ha/test/integration/health_monitor/test_k8s_publisher.py

```diff
@@ -45,22 +43,36 @@ def receive(message):
     try:
         print("********Event Publisher********")
         event_manager = EventManager.get_instance()
-        k8s_filter = K8SFilter()
+        cluster_resource_filter = ClusterResourceFilter()
         component = "hare"
-        resource_type = K8S_ALERT_RESOURCE_TYPE.RESOURCE_TYPE_POD.value
-        state = K8S_ALERT_STATUS.STATUS_FAILED.value
+        resource_type = "node"
+        state = "failed"
         message_type = event_manager.subscribe('hare', [SubscribeEvent(resource_type, [state])])
         print(f"Subscribed {component}, message type is {message_type}")
-        k8s_event = K8SAlert("cortx", "node2", "cortx-data123", K8S_ALERT_STATUS.STATUS_FAILED.value, K8S_ALERT_RESOURCE_TYPE.RESOURCE_TYPE_POD.value, "16215909572")
-
-        timestamp = str(int(time.time()))
-        event_id = timestamp + str(uuid.uuid4().hex)
-        event_type = k8s_event.status
-        if k8s_filter.filter_event(json.dumps(k8s_event.__dict__)):
-            health_event = HealthEvent(event_id, event_type, EVENT_SEVERITIES.CRITICAL.value, "1", "1", "1", "1",
-                            "srvnode_1", "srvnode_1", "pod", "16215909572", "cortx-data-pod", {"namespace": "cortx"})
+        ha_event = {
+                   'header': {
+                       'event_id': "123464",
+                       'timestamp': '234445ww2',
+                   },
+                   'payload': {
+                       'source': "source_1",
+                       'resource_id': "1",
+                       'site_id': 1,
+                       'node_id': 1,
+                       'cluster_id': 1,
+                       'rack_id': 1,
+                       'resource_type': resource_type,
+                       'resource_id': 1,
+                       'resource_status': "failed",
+                       'specific_info': {
+                           'generation_id': '123563',
+                       }
+                   }
+               }
+        if cluster_resource_filter.filter_event(json.dumps(ha_event)):
+            health_event = HealthEvent("source_1", 1, "failed", EVENT_SEVERITIES.CRITICAL.value, "1", "1", "1", "1", "srvnode_1", "srvnode_1", "node", "16215909572", "cortx-data-pod", {"namespace": "cortx"})
```

Comment:
just a suggestion: you can use NOT_DEFINED from ha.fault_tolerance.const for site id, rack id, storage set id